python-docker.spec

```diff
@@ -55,15 +55,15 @@ BuildRequires:  python-backports-ssl_match_hostname
 BuildRequires:  python2-six >= 1.4.0
 BuildRequires:  python-websocket-client >= 0.32.0
 BuildRequires:  python-ipaddress
-BuildRequires:  python2-docker-pycreds
+BuildRequires:  python-docker-pycreds
 %endif  # tests
 
 Requires:       python2-requests
 Requires:       python-websocket-client >= 0.32.0
 Requires:       python2-six >= 1.4.0
 Requires:       python-ipaddress
 Requires:       python-backports-ssl_match_hostname
-Requires:       python2-docker-pycreds
+Requires:       python-docker-pycreds
```

Comment:
version in Extras is actually the same 0.2.1 it's just badly bundled/subpackaged in python-docker-py ...
But let's take this workaround until we figure out better upgrade path.